src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -643,6 +644,13 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
 
         manager.run_passes(func);
     }
+
+    {
+        ov::pass::Manager manager;
```

Comment:
No need to have separate pass manager here. Just register this pass after `ConvertFullyConnectedToFullyConnectedCompressed`

Reply:
Applied it as you commented